src/Our.Umbraco.InnerContent/PropertyEditors/InnerContentPropertyEditor.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Umbraco.Core.PropertyEditors;
+
+namespace Our.Umbraco.InnerContent.PropertyEditors
+{
+    public abstract class InnerContentPropertyEditor : PropertyEditor
```

Comment:
@mattbrailsford Do you think we need this abstract class? I only added it in for "completeness".

Reply:
If it's not doing anything, then I guess not